packages/manager/modules/cloud-connect/src/overview/unlock-port/template.html

```diff
@@ -0,0 +1,22 @@
+<form
+    id="unlockPort"
+    name="unlockPort"
+    data-ng-submit="unlockPort.$valid && $ctrl.unlockPort()"
+    novalidate
+>
+    <oui-modal
+        data-heading="{{:: 'cloud_connect_pop_port_unblock' | translate }}"
+        data-primary-label="{{:: 'cloud_connect_unblock' | translate }}"
+        data-primary-disabled="$ctrl.isLoading"
+        data-secondary-action="$ctrl.goBack()"
+        data-secondary-label="{{:: 'cloud_connect_cancel' | translate }}"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isLoading"
+        data-type="warning"
+    >
+        <p
+            data-translate="cloud_connect_pop_unblock_port_confirm"
+            data-translate-values="{ portNumber: $ctrl.interfaceId }"
+        ></p>
+    </oui-modal>
+</form>
```

Comment:
Why using a `form` here? Also you can call `$ctrl.unlockPort()` in a `primary-action` property